src/refs.c

```diff
@@ -175,10 +177,11 @@ int git_reference_name_to_id(
 	return 0;
 }
 
-static int reference_normalize_for_repo(
+static int reference__normalize_for_repo(
```

Comment:
Sylistic nitpick: we use the double-underscores for things that are internal to the library (but have a scope outside the current file).  That lets us disambiguate public functions.  For example:

* `git_reference_some_function` would be exported and public to libgit2 callers.
* `git_reference__different_function` would be internal to the library

We generally don't do double-underscores for `static`s within a file.  I'm sure there are outliers to this rule, which is not a big deal, but I don't want to add new ones.